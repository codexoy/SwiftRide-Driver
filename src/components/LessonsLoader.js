import React from "react"
import ContentLoader from "rn-content-loader"
import { Circle, Rect } from "react-native-svg"

class LessonsLoader extends React.Component {
	constructor(props) {
		super(props)
		this.width = props.width || 300
		this.height = props.height || 140
	}

	render() {
		const { props } = this

		return (
			<ContentLoader
				rtl
				height={this.height}
				width={this.width}
				speed={1}
				primaryColor="#f3f3f3"
				secondaryColor="#ecebeb"
				{...props}
			>
				<Rect
					x={this.width - 150}
					y="43"
					rx="3"
					ry="3"
					width="85"
					height="6.4"
				/>
				<Rect
					x={this.width - 182}
					y="29"
					rx="4"
					ry="4"
					width="117"
					height="6.4"
				/>
				<Rect x="12" y="36.4" rx="3" ry="3" width="70" height="6.4" />
				<Circle cx={this.width - 24} cy="35.5" r="22.5" />
				<Circle cx={this.width - 24} cy="96.5" r="22.5" />
				<Rect
					x={this.width - 150}
					y="95"
					rx="3"
					ry="3"
					width="85"
					height="6.4"
				/>
				<Rect
					x={this.width - 182}
					y="82"
					rx="4"
					ry="4"
					width="117"
					height="6.4"
				/>
				<Rect x="12" y="88.4" rx="3" ry="3" width="70" height="6.4" />
			</ContentLoader>
		)
	}
}

export default LessonsLoader
