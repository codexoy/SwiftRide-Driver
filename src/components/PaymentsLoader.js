import React from "react"
import ContentLoader from "rn-content-loader"
import { Circle, Rect } from "react-native-svg"

class PaymentsLoader extends React.Component {
	constructor(props) {
		super(props)
	}

	render() {
		const { props } = this

		return (
			<ContentLoader
				rtl
				height={220}
				width={300}
				speed={2}
				primaryColor="#f3f3f3"
				secondaryColor="#ecebeb"
				{...props}
			>
				<Rect x="90" y="16" rx="0" ry="0" width="120" height="49" />
				<Rect x="115" y="76" rx="0" ry="0" width="70" height="10" />
				<Rect x="17" y="110" rx="3" ry="3" width="40" height="6.4" />
				<Rect
					x="180.77"
					y="110"
					rx="4"
					ry="4"
					width="74.88"
					height="6.4"
				/>
				<Circle cx="280" cy="113.5" r="13" />
				<Circle cx="280" cy="175.6" r="13" />
				<Rect
					x="180.77"
					y="172"
					rx="4"
					ry="4"
					width="74.88"
					height="6.4"
				/>
				<Rect x="17" y="172" rx="3" ry="3" width="40" height="6.4" />
			</ContentLoader>
		)
	}
}

export default PaymentsLoader
