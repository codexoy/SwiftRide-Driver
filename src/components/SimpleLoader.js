import React from "react"
import ContentLoader from "rn-content-loader"
import { Rect } from "react-native-svg"

export default class SimpleLoader extends React.Component {
	constructor(props) {
		super(props)
		this.width = props.width || 300
		this.height = props.height || 60
	}

	render() {
		const { props } = this

		return (
			<ContentLoader
				rtl
				height={this.height}
				width={this.width}
				speed={1}
				primaryColor="#f3f3f3"
				secondaryColor="#ecebeb"
				{...props}
			>
				<Rect
					x={this.width - 80}
					y="32"
					rx="4"
					ry="4"
					width="80"
					height="6.4"
				/>
				<Rect x="16" y="32" rx="4" ry="4" width="40.44" height="6.4" />
			</ContentLoader>
		)
	}
}
