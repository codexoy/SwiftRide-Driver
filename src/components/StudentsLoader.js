import React from "react"
import ContentLoader from "rn-content-loader"
import { Circle, Rect } from "react-native-svg"

export default class StudentsLoader extends React.Component {
	constructor(props) {
		super(props)
		this.height = props.height || 220
		this.width = props.width || 340
	}

	render() {
		const { props } = this

		return (
			<ContentLoader
				rtl
				height={this.height}
				width={this.width}
				speed={1}
				primaryColor="#f3f3f3"
				secondaryColor="#ecebeb"
				{...props}
			>
				<Circle cx={this.width - 24} cy="52" r="22.75" />
				<Rect
					x={this.width - 130}
					y="36"
					rx="4"
					ry="4"
					width="70"
					height="6.4"
				/>
				<Rect
					x={this.width - 100}
					y="50"
					rx="3"
					ry="3"
					width="40"
					height="6.4"
				/>

				<Rect
					x={this.width - 100}
					y="64"
					rx="3"
					ry="3"
					width="40"
					height="6.4"
				/>
				<Rect x="14.84" y="40" rx="0" ry="0" width="12" height="10" />
				<Circle cx={this.width - 24} cy="124" r="22.75" />
				<Rect
					x={this.width - 130}
					y="106"
					rx="4"
					ry="4"
					width="70"
					height="6.4"
				/>
				<Rect
					x={this.width - 100}
					y="120"
					rx="3"
					ry="3"
					width="40"
					height="6.4"
				/>

				<Rect
					x={this.width - 100}
					y="134"
					rx="3"
					ry="3"
					width="40"
					height="6.4"
				/>
				<Rect x="14.84" y="110" rx="0" ry="0" width="12" height="10" />
			</ContentLoader>
		)
	}
}
