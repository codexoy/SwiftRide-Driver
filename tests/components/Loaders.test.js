import React from "react"

import renderer from "react-test-renderer"

import SimpleLoader from "../../src/components/SimpleLoader"
import LessonsLoader from "../../src/components/LessonsLoader"
import StudentsLoader from "../../src/components/StudentsLoader"
import PaymentsLoader from "../../src/components/PaymentsLoader"

describe("SimpleLoader", () => {
	test("view renders correctly", () => {
		const tree = renderer
			.create(<SimpleLoader height={100} width={240} />)
			.toJSON()
		expect(tree).toMatchSnapshot()
	})
})

describe("LessonsLoader", () => {
	test("view renders correctly", () => {
		const tree = renderer.create(<LessonsLoader />).toJSON()
		expect(tree).toMatchSnapshot()
	})
})

describe("StudentsLoader", () => {
	test("view renders correctly", () => {
		const tree = renderer.create(<StudentsLoader />).toJSON()
		expect(tree).toMatchSnapshot()
	})
})

describe("PaymentsLoader", () => {
	test("view renders correctly", () => {
		const tree = renderer.create(<PaymentsLoader />).toJSON()
		expect(tree).toMatchSnapshot()
	})
})
