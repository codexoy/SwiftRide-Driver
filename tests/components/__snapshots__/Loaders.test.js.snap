// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LessonsLoader view renders correctly 1`] = `
<View>
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="85"
    x={150}
    y="43"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="117"
    x={118}
    y="29"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="70"
    x="12"
    y="36.4"
  />
  <RNSVGCircle
    cx={276}
    cy="35.5"
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    r="22.5"
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  />
  <RNSVGCircle
    cx={276}
    cy="96.5"
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    r="22.5"
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="85"
    x={150}
    y="95"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="117"
    x={118}
    y="82"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="70"
    x="12"
    y="88.4"
  />
</View>
`;

exports[`PaymentsLoader view renders correctly 1`] = `
<View>
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="49"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="0"
    ry="0"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="120"
    x="90"
    y="16"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="10"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="0"
    ry="0"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="70"
    x="115"
    y="76"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="40"
    x="17"
    y="110"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="74.88"
    x="180.77"
    y="110"
  />
  <RNSVGCircle
    cx="280"
    cy="113.5"
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    r="13"
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  />
  <RNSVGCircle
    cx="280"
    cy="175.6"
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    r="13"
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="74.88"
    x="180.77"
    y="172"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="40"
    x="17"
    y="172"
  />
</View>
`;

exports[`SimpleLoader view renders correctly 1`] = `
<View>
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="80"
    x={160}
    y="32"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="40.44"
    x="16"
    y="32"
  />
</View>
`;

exports[`StudentsLoader view renders correctly 1`] = `
<View>
  <RNSVGCircle
    cx={316}
    cy="52"
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    r="22.75"
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="70"
    x={210}
    y="36"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="40"
    x={240}
    y="50"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="40"
    x={240}
    y="64"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="10"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="0"
    ry="0"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="12"
    x="14.84"
    y="40"
  />
  <RNSVGCircle
    cx={316}
    cy="124"
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    r="22.75"
    rotation={0}
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    x={0}
    y={0}
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="4"
    ry="4"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="70"
    x={210}
    y="106"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="40"
    x={240}
    y="120"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="6.4"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="3"
    ry="3"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="40"
    x={240}
    y="134"
  />
  <RNSVGRect
    fill={
      Array [
        0,
        4278190080,
      ]
    }
    fillOpacity={1}
    fillRule={1}
    height="10"
    matrix={
      Array [
        1,
        0,
        0,
        1,
        0,
        0,
      ]
    }
    opacity={1}
    originX={0}
    originY={0}
    propList={Array []}
    rotation={0}
    rx="0"
    ry="0"
    scaleX={1}
    scaleY={1}
    skewX={0}
    skewY={0}
    stroke={null}
    strokeDasharray={null}
    strokeDashoffset={null}
    strokeLinecap={0}
    strokeLinejoin={0}
    strokeMiterlimit={4}
    strokeOpacity={1}
    strokeWidth={1}
    vectorEffect={0}
    width="12"
    x="14.84"
    y="110"
  />
</View>
`;
